src/main/java/frc/utils/FileHandler.java

```diff
@@ -0,0 +1,25 @@
+package frc.utils;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+    public class FileHandler {
```

Comment:
```suggestion
    public class CMDHandler{
```
my suggestion is not a good name is think but change the name to something in this direction